logger/tests/TestLogger.py

```diff
@@ -0,0 +1,24 @@
+from logbook import Logger
+import unittest
+#from ../Logger import Logger
```

Comment:
I'd remove this line just to keep things clean, unless it's kept in here for some reason.